### PR TITLE
rgw: Remove the useless output when list zones

### PR DESCRIPTION
**Pull request by iliul**
```
root@9d02aec4a204:~/ceph-12.2.0/build# radosgw-admin zone list  
2017-09-03 08:43:11.810813 7f64868aecc0 -1 WARNING: all dangerous and experimental features are enabled.
2017-09-03 08:43:11.811122 7f64868aecc0 -1 WARNING: all dangerous and experimental features are enabled.
2017-09-03 08:43:11.813236 7f64868aecc0 -1 WARNING: all dangerous and experimental features are enabled.
{
    "default_info": "",
    "zones": [
        "default"
    ]
}
              //delete it
root@9d02aec4a204:~/ceph-12.2.0/build#
```

### CODE REVIEW

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -4115,7 +4115,6 @@ int main(int argc, const char **argv)
 	encode_json("zones", zones, formatter);
 	formatter->close_section();
 	formatter->flush(cout);
-	cout << std::endl;
```
Comment (joscollin) [inline src/rgw/rgw_admin.cc]: If so what about the other instances of `cout << std::endl;` in the same `switch` ? I don't think it is intentional, as it always returns `0` after the break. So you can remove them too.

Comment (iliul) [reply src/rgw/rgw_admin.cc]: @joscollin done!

File: src/rgw/rgw_admin.cc [hunk 2]
```diff
@@ -5012,7 +4994,6 @@ int main(int argc, const char **argv)
     }
     formatter->close_section();
     formatter->flush(cout);
-    cout << std::endl;
```
Comment (joscollin) [inline src/rgw/rgw_admin.cc]: ditto

File: src/rgw/rgw_admin.cc [hunk 3]
```diff
@@ -5103,7 +5084,6 @@ int main(int argc, const char **argv)
       }
       formatter->close_section();
       formatter->flush(cout);
-      cout << std::endl;
```
Comment (joscollin) [inline src/rgw/rgw_admin.cc]: ditto

File: src/rgw/rgw_admin.cc [hunk 4]
```diff
@@ -5154,7 +5134,6 @@ int main(int argc, const char **argv)
     }
     formatter->close_section();
     formatter->flush(cout);
-    cout << std::endl;
```
Comment (joscollin) [inline src/rgw/rgw_admin.cc]: ditto

File: src/rgw/rgw_admin.cc [hunk 5]
```diff
@@ -6941,7 +6920,6 @@ int main(int argc, const char **argv)
     }
     encode_json("bounds", bounds, formatter);
     formatter->flush(cout);
-    cout << std::endl;
```
Comment (joscollin) [inline src/rgw/rgw_admin.cc]: ditto

Comment (iliul) [reply src/rgw/rgw_admin.cc]: reverted!

File: src/rgw/rgw_admin.cc (hunk 4 above)
Comment (iliul) [reply src/rgw/rgw_admin.cc]: reverted

File: src/rgw/rgw_admin.cc (hunk 3 above)
Comment (iliul) [reply src/rgw/rgw_admin.cc]: revert

File: src/rgw/rgw_admin.cc (hunk 2 above)
Comment (iliul) [reply src/rgw/rgw_admin.cc]: reverted

File: src/rgw/rgw_admin.cc [hunk 6]
```diff
@@ -994,7 +994,6 @@ static void show_user_info(RGWUserInfo& info, Formatter *formatter)
 {
   encode_json("user_info", info, formatter);
   formatter->flush(cout);
-  cout << std::endl;
```
[the remark this answers is not in the document]
Comment (iliul) [reply src/rgw/rgw_admin.cc]: @joscollin ok，I had reverted it!

**Comment by iliul** [comment]
@joscollin could you help review this pr again, thanks

**Pull request merged**